charts/verdaccio/templates/deployment.yaml

```diff
@@ -66,6 +67,11 @@ spec:
             - mountPath: /verdaccio/storage
               name: storage
               readOnly: false
+            {{- if .Values.useSecretHtpasswd }}
+            - mountPath: /verdaccio/auth
+              name: htpasswd
+              readOnly: true
```

Comment:
Using [`subPath`](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath) like this
```yaml
- mountPath: /verdaccio/storage/htpasswd
  name: htpasswd
  subPath: htpasswd
  readOnly: true
```
 will mount a single file instead of a directory meaning you can nest the mount within volumes.

Reply:
Will do :)